.github/workflows/reusable-windows-msi.yml

```diff
@@ -0,0 +1,24 @@
+name: TestsMSI
+
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: CPU architecture
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: installer for ${{ inputs.arch }}
+    runs-on: windows-latest
+    timeout-minutes: 60
+    env:
+      IncludeFreethreaded: true
```

Comment:
@hugovk I kept this here as it shouldn't have any effect but would make any future backports apply cleaner.